recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: conda-forge-build-setup
-  version: 2.1.0
+  version: 2.2.0
```

Comment:
Bumped the version number as the incorrect comments are resulting in [noisy warnings]( https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.3504/job/5mectb42afsdy380#L167 ). Fixing the comments and bumping the version number should eliminate that noise.